Resources/Prototypes/Entities/Structures/Machines/vending_machines.yml

```diff
@@ -1358,6 +1358,42 @@
     energy: 1.6
     color: "#d4ab33"
 
+- type: entity
+  parent: VendingMachine
+  id: VendingMachineQuickThreads
+  name: QuickThreads
+  description: A vending machine for clothing.
+  components:
+  - type: VendingMachine
+    pack: QuickThreadsInventory
+    dispenseOnHitChance: 0.25
+    dispenseOnHitThreshold: 2
+    offState: off
+    brokenState: broken
+    normalState: normal-unshaded
+    ejectState: eject-unshaded
+    denyState: deny-unshaded
+    screenState: screen
```

Comment:
```suggestion

```
idk the point of this but it's what's causing the fail

Reply:
i copied the coffee machine prototype which has a screen state L